.gitattributes

```diff
@@ -0,0 +1 @@
+screenshots/** filter=lfs diff=lfs merge=lfs -text
```

Comment:
🎉 Nice, this should keep us from accidentally committing images to the git history